README.md

```diff
@@ -57,7 +57,7 @@ If you want to make changes to the `dind` image being used, make your changes to
 ### How can I connect to a published port from the outside world?
 
 
-If you need to access your services from outside, use the following URL pattern `http://pwd<hyphen-ip>-<port>.<host#>.labs.play-with-docker.com` (i.e: http://pwd10-2-135-3-80.host3.labs.play-with-docker.com/).
+If you need to access your services from outside, use the following URL pattern `http://pwd<hyphen-ip>-<port>.<host#>.labs.play-with-docker.com` (i.e: http://<i></i>pwd10-2-135-3-80.host3.labs.play-with-docker.com).
```

Comment:
I see a <i></i> that's not enclosing anything?

Reply:
It isn't, it just removes the hyperlink. I went to read the page and clicked on the http://pwd<hyphen-ip>-<port>.<host#>.labs.play-with-docker.com because I thought it linked to a resource. This will break it from being a hyperlink.

Reply:
:+1: